contracts/libraries/FixedPoint.sol

```diff
@@ -78,46 +78,49 @@ library FixedPoint {
         uint224 uppero_lowers = uint224(upper_other) * lower_self; // * 2^-112
 
         // so the bit shift does not overflow
-        require(upper <= uint112(-1), 'FixedPoint: MULUQ_OVERFLOW_UPPER');
+        require(upper <= uint112(-1), 'FixedPoint::muluq: upper overflow');
 
         // this cannot exceed 256 bits, all values are 224 bits
         uint256 sum = uint256(upper << RESOLUTION) + uppers_lowero + uppero_lowers + (lower >> RESOLUTION);
 
         // so the cast does not overflow
-        require(sum <= uint224(-1), 'FixedPoint: MULUQ_OVERFLOW_SUM');
+        require(sum <= uint224(-1), 'FixedPoint::muluq: sum overflow');
 
         return uq112x112(uint224(sum));
     }
 
     // divide a UQ112x112 by a UQ112x112, returning a UQ112x112
     function divuq(uq112x112 memory self, uq112x112 memory other) internal pure returns (uq112x112 memory) {
-        require(other._x > 0, 'FixedPoint: DIV_BY_ZERO_DIVUQ');
+        require(other._x > 0, 'FixedPoint::divuq: division by zero');
         if (self._x == other._x) {
             return uq112x112(uint224(Q112));
         }
         if (self._x <= uint144(-1)) {
             uint256 value = (uint256(self._x) << RESOLUTION) / other._x;
-            require(value <= uint224(-1), 'FixedPoint: DIVUQ_OVERFLOW');
+            require(value <= uint224(-1), 'FixedPoint::divuq: division overflow');
             return uq112x112(uint224(value));
         }
 
         uint256 result = FullMath.mulDiv(Q112, self._x, other._x);
-        require(result <= uint224(-1), 'FixedPoint: DIVUQ_OVERFLOW');
+        require(result <= uint224(-1), 'FixedPoint::divuq: division overflow');
         return uq112x112(uint224(result));
     }
 
     // returns a UQ112x112 which represents the ratio of the numerator to the denominator
     // lossy
     function fraction(uint112 numerator, uint112 denominator) internal pure returns (uq112x112 memory) {
-        require(denominator > 0, 'FixedPoint: DIV_BY_ZERO_FRACTION');
-        return uq112x112((uint224(numerator) << RESOLUTION) / denominator);
+        require(denominator > 0, 'FixedPoint::fraction: division by zero');
+        uint256 result = (uint256(numerator) << RESOLUTION) / denominator;
+        require(result <= uint224(-1), 'FixedPoint::fraction: division overflows');
```

Comment:
we don't need this, even if the numerator is `uint112(-1)`, multiplying by `2**112` and dividing by 1 is still < `2**224-1`